apigee_m10n.permissions.yml

```diff
@@ -7,6 +7,9 @@ access monetization packages:
 view mint prepaid reports:
   title: 'View Mint Prepaid Reports'
 
+refresh prepaid balance reports:
```

Comment:
I don't think we need a full permission for this, do we?  If you have permission to view balance, you should be able to refresh the balance also.

Reply:
It might be handy. Site admins could disable the feature by not granting anyone the permission to refresh.

Reply:
@cnovak @Jaesin can you let me know if we need refresh permissions or not please? I'm doing a refactor. If we are not adding permissions I can remove it while I'm at it. Thanks.

Reply:
You can leave it in since @Jaesin said it may be good, thx.  Just wanted to ask to make sure it made sense.